src/reverse/RTTIHelper.cpp

```diff
@@ -520,43 +543,64 @@ sol::variadic_results RTTIHelper::ExecuteFunction(RED4ext::CBaseFunction* apFunc
     ResetAllocator ___allocatorReset;
 
     std::vector<RED4ext::CStackType> callArgs(apFunc->params.size);
```

Comment:
std::vector should be replaced with TiltedPhoques::Vector, it's an overload that will also use the pushed allocator, so you will get free memory allocations here

Reply:
I was using `TiltedPhoques::Vector` when started the RTTI helper and then switched to `std::vector` because in my tests the last one showed surprisingly better performance. Let me do the tests again in the final version.

Reply:
Fixed.